integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala

```diff
@@ -39,14 +42,15 @@ object CarbonThriftServer {
 
     val sparkHome = System.getenv.get("SPARK_HOME")
     if (null != sparkHome) {
-      builder.config("carbon.properties.filepath",
-        sparkHome + '/' + "conf" + '/' + "carbon.properties")
-      System.setProperty("carbon.properties.filepath",
-        sparkHome + '/' + "conf" + '/' + "carbon.properties")
+      val file = new File(sparkHome + '/' + "conf" + '/' + "carbon.properties")
+      if (file.exists()) {
+        builder.config("carbon.properties.filepath", file.getCanonicalPath)
+        System.setProperty("carbon.properties.filepath", file.getCanonicalPath)
+      }
     }
-    CarbonProperties.getInstance().addProperty("carbon.storelocation", args.head)
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.STORE_LOCATION, args.head)
 
```

Comment:
CarbonCommonConstants.STORE_LOCATION also get value from carbon.properties.carbon.storelocation, why need do this change ?